src/StyledLink/index.js

```diff
@@ -1,50 +1,22 @@
 import React from 'react'
-import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
-import {
-  faAngleRight,
-  faArrowRight,
-  faAngleDown,
-  faDownload,
-  faLock,
-  faAngleDoubleRight, // Needs a double replacement.
-  faVideo
-} from '@fortawesome/free-solid-svg-icons'
-import styles from './index.module.css'
```

Comment:
Removing all of this and will recreate it with hero icons later.
Also, thing to note, jest doesn't like css modules and we'd have to add extra support for that.